bundle/Enums/HandlerType.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Netgen\Bundle\IbexaScheduledVisibilityBundle\Enums;
+
+enum HandlerType: string
```

Comment:
I think this can be removed now?

Reply:
removed in 0c4965765038af03a99b0eabe43e6dcd1ab83133